hack/build/docker/cdi-func-test-file-host-http/Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM fedora:28
+FROM fedora:29
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
 ENV container docker
 
-RUN dnf -y update && dnf -y install nginx && dnf clean all -y
+COPY fedora.repo /tmp/fedora_ci.repo
+
+RUN mv /etc/yum.repos.d/fedora.repo /etc/yum.repos.d/fedora.repo.org && \
+    mv /tmp/fedora_ci.repo /etc/yum.repos.d/fedora.repo && \
```

Comment:
generally this is seems like the right technique for using the repo file but id recommend using a single file for all repos and basic dnf configuration and replacing the `/etc/fnd/dnf.conf` file with it that way you know you have full control over the repos in the container.

The file should also contain a line like the following in the `[main]` section:

    reposdir = /dev/null

This disables the `*.repos.d` directories, so you know the file fully controls what `dnf` does.